.coverage.sh

```diff
@@ -8,23 +8,33 @@ mkdir -p $COVERAGE_DIR
 pushd $COVERAGE_DIR
 if [ -z "$KEEP" ]; then trap "popd; rm -rf $COVERAGE_DIR" EXIT; fi
 
+# copy over everything
 $(which cp) -r ../* .
 
+# prepare the environment
+eval `opam config env`
 opam install bisect_ppx ocveralls -y
 
+# run the tests with coverage enabled
 export BISECT_ENABLE=YES
 jbuilder runtest
 
-outs=$(find . | grep bisect.*.out)
-bisect-ppx-report -I $(dirname $outs[1]) -text report $outs
-bisect-ppx-report -I $(dirname $outs[1]) -summary-only -text summary $outs
-if [ -n "$HTML" ]; then bisect-ppx-report -I $(dirname $outs[1]) -html ../html-report $outs; fi
+# move all bisect files here
+find . -name 'bisect*.out' -exec mv {} . \;
+
+DIRECTORIES=$(find _build/default -type d -not -path '*/\.*')
```

Comment:
Not sure what this is meant to exclude, are there some hidden directories under _build?

Reply:
Yes, .ppx with the rewriters. It is not necessary though, just nicer output